regparser/web/jobs/models.py

```diff
@@ -25,3 +33,28 @@ class ParsingJob(models.Model):
         ("complete_with_errors", "complete_with_errors")
     ), default="received")
     url = models.URLField(blank=True, max_length=2000)
+
+    def save(self, *args, **kwargs):
+        super(ParsingJob, self).save(*args, **kwargs)
+
+
+class PipelineJob(ParsingJob):
+
+    cfr_title = models.IntegerField()
+    cfr_part = models.IntegerField()
+    only_latest = models.BooleanField(default=False)
+
+
+class ProposalPipelineJob(ParsingJob):
+
+    file_hexhash = models.CharField(max_length=32)
+    only_latest = models.BooleanField(default=True)
+
+
+class RegulationFile(models.Model):
+
+    contents = models.BinaryField()
+    file = models.FileField(null=True)
```

Comment:
I wouldn't think we'd need _both_ a `contents` and a `file` field. Ideally, we'd get rid of the blob and only access the via a [storage](https://docs.djangoproject.com/en/1.10/ref/files/storage/) system. This would allow us to swap out the storage system (e.g. file system for local stuff, s3 for production) and would obviate other issues like unintentionally deserializing the whole file when loading a model from disk.